receiver/windowsperfcountersreceiver/README.md

```diff
@@ -58,17 +55,45 @@ windowsperfcounters:
           recreate_query: <true or false>
 ```
 
-*Note `instances` can have several special values depending on the type of
-counter:
+> [!NOTE]
+> The `instances` can have several special values depending on the type of counter:
 
 Value | Interpretation
 -- | --
 Not specified | This is the only valid value if the counter has no instances
 `"*"` | All instances
-`"_Total"` | The "total" instance
+`"_Total"` | The "total" instance, that aggregates the values of all other instances. See below for its special treatment.
 `"instance1"` | A single instance
 `["instance1", "instance2", ...]` | A set of instances
-`["_Total", "instance1", "instance2", ...]` | A set of instances including the "total" instance
+
+> [!NOTE]
+> Aggregation counter and the behavior of the `_Total` instance
+
+The `_Total` must be collected individually on its own metric,
+since it is dropped when collected together with other instances.
+
+```yaml
+windowsperfcounters:
+  metrics:
+    processor.time.total:
+      description: Total CPU active and idle time
+      unit: "%"
+      gauge:
+  collection_interval: 30s
+  perfcounters:
+    - object: "Processor"
+      instances:
+          - "_Total"
+      counters:
+        - name: "% Processor Time"
+          metric: processor.time.total
+```
+
+> [!WARNING]
```

Comment:
This actually gives an idea to how better handle counters that don't use `_Total` for their aggregation. We can have a configuration called something like `aggregation_name`, that by default is `_Total`, if someone is using a counter that uses a different name for the aggregation instance we let them specify so the problem of double counting can be easily solved.

Reply:
The aggregation_name would be used to decide which instance to drop from a wildcard collection? That makes sense to me.

Reply:
Yes, so it keeps the current default behavior, but, if some user wants to drop the `_Global_` they add that to the configuration.